spyder/utils/programs.py

```diff
@@ -91,6 +91,10 @@ def is_program_installed(basename, extra_paths=[]):
     ):
         return basename
 
+    pixi = [
+        osp.join(home, '.pixi', 'bin'),
+    ]
```

Comment:
```suggestion
    pixi = [osp.join(home, '.pixi', 'bin')]
```

No need to have this statement in multiple lines.